main/index.ts

```diff
@@ -133,6 +136,12 @@ app.on('will-finish-launching', () => {
   });
 });
 
-app.on('quit', () => {
-  cleanPastRecordings();
+app.on('before-quit', async (event: any) => {
+  if (!onExitCleanupComplete) {
+    event.preventDefault();
+    await stopRecordingWithNoEdit();
```

Comment:
when i added the quit option to the context menu during recording, i noticed that when you quit in the middle of a recording, an error would come up the next time you opened the app. I added this to make sure the recording is stopped and cleaned up before quitting. not sure if this is the best way to do it though.